tests/typecheck/fields/test_related.yml

```diff
@@ -1502,14 +1502,36 @@
 -   case: test_reverse_m2m_relation_checks_other_model
     main: |
         from myapp.models import Author
+        # With builtin manager/queryset
         Author().book_set.filter(featured=True)
         Author().book_set.filter(xyz=True)  # E: Cannot resolve keyword 'xyz' into field. Choices are: authors, featured, id  [misc]
+        Author().book_set.values_list("featured")
```

Comment:
Can we see what type will be resolved under `featured`? I expect it to be `bool` (not `Any` or something).

Reply:
Yeah, sure

Reply:
Done